src/device/arm/arm-cortex-a53.S

```diff
@@ -0,0 +1,31 @@
+
+.section ".text.boot"
+
+.global _start
+
+_start:
+    // read cpu id, stop slave cores
+    mrs     x1, mpidr_el1
+    and     x1, x1, #3
+    cbz     x1, 2f
+    // cpu id > 0, stop
+1:  wfe
+    b       1b
+2:  // cpu id == 0
+
+    // set stack before our code
+    ldr     x1, =_start
+    mov     sp, x1
+
+    // clear bss
+    //ldr     x1, =__bss_start
+    //ldr     w2, =__bss_size
+3:  //cbz     w2, 4f
+    //str     xzr, [x1], #8
+    //sub     w2, w2, #1
+    //cbnz    w2, 3b
```

Comment:
Why did you comment this out? I think it is necessary. If it's because of problems with the linker, you need to rename `__bss_start` to `_sbss` and `__bss_end` to `_ebss`, because those are the names that are used in the linker script.
You can also do this in Go code and in fact there is a function `preinit` which does that, but it doesn't seem to be called from anywhere.

Reply:
Fixed. Eliminated dead code in asm, then moved to using the go code that is already in use by other runtimes (risc v, avr, etc).